<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>609</width>
    <height>685</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>609</width>
    <height>685</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>609</width>
    <height>685</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MMMLauncher</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/images/images/epi.ico</normaloff>:/images/images/epi.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="minimumSize">
    <size>
     <width>609</width>
     <height>685</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>609</width>
     <height>685</height>
    </size>
   </property>
   <widget class="QLabel" name="labelUpdate">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>627</y>
      <width>231</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Für ein Update auf &quot;Update&quot; klicken!</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>310</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>6</number>
     </property>
     <item>
      <widget class="QPushButton" name="buttonDownload">
       <property name="text">
        <string>Download</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonStart">
       <property name="text">
        <string>Starten</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonSetup">
       <property name="text">
        <string>Setup</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>590</y>
      <width>261</width>
      <height>33</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="topMargin">
      <number>9</number>
     </property>
     <item>
      <widget class="QPushButton" name="buttonUpdate">
       <property name="text">
        <string>Update</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonDownloadall">
       <property name="text">
        <string>Alle Laden</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonAbout">
       <property name="text">
        <string>Über...</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>646</y>
      <width>260</width>
      <height>23</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>260</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>26</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QGraphicsView" name="epiScreenshoot">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>320</width>
      <height>200</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>320</width>
      <height>200</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>320</width>
      <height>200</height>
     </size>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>350</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Zusätzliche Dateien:</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>590</width>
      <height>267</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>590</width>
      <height>267</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>590</width>
      <height>267</height>
     </size>
    </property>
    <property name="showGrid">
     <bool>false</bool>
    </property>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
   </widget>
   <widget class="QPlainTextEdit" name="epiDescription">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>530</y>
      <width>320</width>
      <height>139</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>320</width>
      <height>139</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>316</width>
      <height>139</height>
     </size>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="centerOnScroll">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QTreeWidget" name="directoryContent">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>370</y>
      <width>260</width>
      <height>222</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>260</width>
      <height>222</height>
     </size>
    </property>
    <property name="expandsOnDoubleClick">
     <bool>false</bool>
    </property>
    <attribute name="headerVisible">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string notr="true">1</string>
     </property>
    </column>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>591</width>
      <height>301</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="tabsClosable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="allEpisodes">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QTreeWidget" name="treeWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>591</width>
        <height>261</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="indentation">
       <number>15</number>
      </property>
      <property name="rootIsDecorated">
       <bool>false</bool>
      </property>
      <property name="itemsExpandable">
       <bool>true</bool>
      </property>
      <property name="expandsOnDoubleClick">
       <bool>false</bool>
      </property>
      <attribute name="headerStretchLastSection">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Titel</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Autor</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
